src/liquid/components/ld-input/ld-input.tsx

```diff
@@ -378,6 +378,13 @@ export class LdInput implements InnerFocusable, ClonesAttributes {
         <Host class={cl} onClick={this.handleClick}>
           <textarea
             {...clonedAttributesWithoutType}
+            aria-disabled={
+              this.disabled || isAriaDisabled(this.ariaDisabled)
+                ? 'true'
+                : undefined
+            }
```

Comment:
Why do you want to set `aria-disabled`, when `disabled` is set, even if `aria-disabled` itself may not be set? And if you now already check of non-false values of `aria-disabled`, why not just pass `this.ariaDisabled` 1:1?

Reply:
Regarding your first question: I think we did it this way to make sure styles which use the `[aria-disabled]` selector work in that case as well. I guess we didn't want to write a double selector all over the place in the CSS file.

Regarding your second question: Pretty similar to the first one. We kind of make sure the selector is consistent, no matter what the component consumer sets as a value. It can be `"yolo"` , which would make the input _dimmed_ to the screen reader, but the CSS selector wouldn't work.

There is still an issue which I must fix before merging this PR. The selector `[aria-disabled]:not([aria-disabled='false'])` is wrong. It selects an element with a non-value attribute `aria-disabled`, however the element would not be recognized as _dimmed_ by a screen reader in that case.